fms/models/__init__.py

```diff
@@ -208,6 +206,19 @@ def _is_dp(distributed_strategy):
     return distributed_strategy in {"fsdp", "hsdp", "ddp"}
 
 
+def _load_state_dict_into_model(
+    model, state_dict, device, rank, world_size, checkpoint_format
+):
+    if checkpoint_format == "st":
+        # In this case state_dict contains info about what files to open and how to
+        # match FMS weights to original weight names
```

Comment:
I don't quite understand this comment. it looks like we're still using `serialization.get_adapted` before calling this and not sure where an HF st state_dict would have got the info to convert to fms weight names? maybe a stale comment from a previous revision?